skimage/filters/_frangi.py

```diff
@@ -0,0 +1,186 @@
+u"""Implemnetation of Frangi and Hessian filters support.
```

Comment:
It's probably a nonbreaking space. You must be able to highlight non-ascii chars with your text editor.